src/ngraph/frontend/onnx_import/op/lp_norm.cpp

```diff
@@ -39,11 +39,17 @@ namespace ngraph
                 NodeVector lp_norm(const Node& node)
                 {
                     const std::shared_ptr<ngraph::Node> data{node.get_ng_inputs().at(0)};
+                    const auto data_shape = data->get_output_partial_shape(0);
```

Comment:
Please consider renaming variable to `data_pshape`.

Reply:
as above